packages/api/src/command/medical/patient/consolidated-create.ts

```diff
@@ -83,15 +82,17 @@ const convertCollectionBundleToTransactionBundle = ({
     )
       continue;
 
+    const contained = resource.contained ? [...resource.contained, patient] : [patient];
     const transactionEntry: BundleEntry = {
       resource: {
         ...resource,
-        contained: resource.contained ? [...resource.contained, patient] : [patient],
+        contained,
       },
-      request: resource.id
-        ? { method: "PUT", url: resource.resourceType + "/" + resource.id }
-        : { method: "POST", url: resource.resourceType },
+      request: { method: "PUT", url: resource.resourceType + "/" + resource.id },
```

Comment:
Is there always a resourceId and thats why we removed the POST?

Reply:
Yes. The flow upstream is such that the CX `MUST` provide IDs. Otherwise, we throw errors.